cfspopcon/formulas/inductances.py

```diff
@@ -0,0 +1,729 @@
+    ),
+    return_units=dict(internal_inductance=ureg.henry),
+    pass_as_kwargs=("internal_inductance_geometry",),
+)
+def calc_internal_inductance(
+    major_radius: float,
+    internal_inductivity: float,
+    plasma_volume: float = 0 * ureg.meter**3,
+    poloidal_circumference: float = 0 * ureg.meter,
+    internal_inductance_geometry: InternalInductanceGeometry = InternalInductanceGeometry.Cylindrical,
+) -> Unitfull:
+    """Calculate the internal inductance of the plasma (assuming a circular cross-section).
+
+    A power-balance model for local helicity injection startup in a spherical tokamak :cite:`Barr_2018`
+
+    Args:
+        plasma_volume: [m^3] : term:`glossary link<plasma_volume>`
+        poloidal_circumference: [m] :term:`glossary link<poloidal_circumference>`
+        major_radius: [m] :term:`glossary link<major_radius>`
+        plasma_current: [A] :term:`glossary link<plasma_current>`
+        magnetic_field_on_axis: [T] :term:`glossary link<magnetic_field_on_axis>`
+        minor_radius: [m] :term:`glossary link<minor_radius>`
+        internal_inductivity: [~] :term:`glossary link<internal_inductivity>`
+        internal_inductance_geometry : [] :term:`glossary link<internal_inductance_geometry>`
+
+    Returns:
+        [henry] :term:`internal_inductance`
+    """
+    internal_inductance = constants.mu_0 * major_radius * internal_inductivity / 2
+
+    if internal_inductance_geometry == InternalInductanceGeometry.NonCylindrical:
+        internal_inductance = constants.mu_0 * internal_inductivity * plasma_volume / (poloidal_circumference**2)
+
+    return float(internal_inductance)
+
+
+@wraps_ufunc(
+    input_units=dict(
+        inverse_aspect_ratio=ureg.dimensionless,
+        areal_elongation=ureg.dimensionless,
+        beta_poloidal=ureg.dimensionless,
+        major_radius=ureg.m,
+        internal_inductivity=ureg.dimensionless,
+        surface_inductance_coefficients=None,
+    ),
+    return_units=dict(external_inductance=ureg.henry),
+    pass_as_kwargs=("surface_inductance_coefficients",),
+)
+def calc_external_inductance(
+    inverse_aspect_ratio: float,
+    areal_elongation: float,
+    beta_poloidal: float,
+    major_radius: float,
+    internal_inductivity: float,
+    surface_inductance_coefficients: SurfaceInductanceCoeffs,
+) -> Unitfull:
+    """Calculate the external self-inductance of the plasma for which the current-induced surface flux of the plasma is generated from eq. 13 in...
+
+    A power-balance model for local helicity injection startup in a spherical tokamak :cite:`Barr_2018`
+
+    Args:
+        inverse_aspect_ratio: [~] :term:`glossary link<inverse_aspect_ratio>`
+        areal_elongation: [~] :term:`glossary link<areal_elongation>`
+        beta_poloidal: [~] :term:`glossary link<beta_poloidal>`
+        internal_inductivity: [~] :term:`glossary link<internal_inductivity>`
+        major_radius: [m] :term:`glossary link<major_radius>`
```

Comment:
just switching order to match function signature
```suggestion
        major_radius: [m] :term:`glossary link<major_radius>`
        internal_inductivity: [~] :term:`glossary link<internal_inductivity>`
```

Reply:
@IsaacSavona, these are stil in the wrong order